packages/oak/lib/core/Editable/index.js

```diff
@@ -21,37 +25,24 @@ export default forwardRef(({
 }, ref) => {
   const { _settingsHolderRef, overrides, oakRef } = useBuilder();
   const options = useOptions();
-  const [popper, setPopper] = useState();
-  const [reference, setReference] = useState();
-
   const [state, dispatch] = useReducer(mockState, {
     opened: false,
   });
 
+  const { x, y, reference, floating, strategy, refs } = useFloating({
+    whileElementsMounted: autoUpdate,
+    middleware: [
+      offset({ mainAxis: 5 }),
+      autoPlacement({
+        boundary: oakRef?.current,
+      }),
```

Comment:
I think you missed the `allowedPlacements` prop here

Reply:
I left it out on purpose, I assumed that as long as it was bound to the root boundary, the placement was not important

Reply:
Mmmmh but it implies that the catalog can be on the side of the button which is weird (for me) 

https://user-images.githubusercontent.com/10706836/189898948-fbc1c680-2ed2-4a59-91f7-95b93d70f72f.mov